pkg/pidfile/pidfile_test.go

```diff
@@ -0,0 +1,31 @@
+package pidfile
+
+import (
+	"testing"
+)
+
+func TestCreateFile(t *testing.T) {
+	_, err := New("/tmp/testfile")
```

Comment:
consider using filepath.Join() for platform-independent paths. And also `ioutil.TempDir(os.TempDir(), "test")` to not to hardcode `/tmp`